apps/aeutils/priv/extensions/cli.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
this is "okish" but I wonder if it's possible to integrate as subcommand (or whatever is the term) so that I can run it with `bin/aeternity cli` or `bin/aeternity admin` or whatever we decide ?

Reply:
nevermind, this is indeed what the implementation does :)